src/test/java/mops/module/services/ModulServiceDatabaseTest.java

```diff
@@ -178,6 +182,11 @@ public void approveModulModificationAntragTest() {
     public void completeAddModulRoutineTest() {
 
         Modul modul = JsonService.jsonObjectToModul(completeModul);
+
+        Set<String> modulbeauftragte = new HashSet<>();
+        modulbeauftragte.add("Michael Schöttner");
+        modul.setModulbeauftragte(modulbeauftragte);
```

Comment:
Warum wird hier der Modulbeauftragte erneut eingefügt?  In Zeile 95 steht die Information doch schon im Json, oder?